### PR TITLE
Update metatransactions-gsn.md to discuss GSNv1 and GSNv2

### PR DESCRIPTION
**Pull request by JUDOKICK**
Added and extended docs to include deployed GSNv1 that is deployed in matic testnet and mainnet and links to help new users onboard

### CODE REVIEW

File: docs/develop/metatransactions/metatransactions-gsn.md [hunk 1]
```diff
@@ -16,85 +16,120 @@ That's where GSN comes into picture with an interesting proposal for improving D
...
+
+Then you simply fund the ```RelayHub.sol``` with your contracts deployed address using the recipient funder function ```depositFor(address ContractToFund)``` or using the [OpenGSN Funder Tool](https://www.opengsn.org/recipients)
+
+#### Approval Strategies
+
+In GSNv1 there were a couple of various approval strategies for deciding which function calls to relay and which ones not to. The primary ones are discussed by [OpenZeppelin](https://docs.openzeppelin.com/contracts/3.x/gsn-strategies) although you are free to implement your own in the ```approveRelayedCall``` function.
+
+#### Deploying your own Relayer
+
+For those looking to run your own relayer, the folks at [Creol.io](Creol.io) have created a very simple guide and tutorial for doing so on one of their github pages. [Running your own GSNv1 Relayer](https://github.com/JUDOKICK/matic-gsn-relayer-creol). It includes a simple to use Docker container script that is based on xDai's image for this.
```
Comment (itzmeanjan) [inline docs/develop/metatransactions/metatransactions-gsn.md:50]: Can you please remove last sentence from this paragraph ? 

@JUDOKICK

Comment (itzmeanjan) [inline docs/develop/metatransactions/metatransactions-gsn.md:50]: Also please replace `folks at Creol.io` with `our friends at Creol.io`. Sounds better, right @JUDOKICK ?

Comment (itzmeanjan) [reply docs/develop/metatransactions/metatransactions-gsn.md:50]: This one is not required here.

**Review by itzmeanjan — CHANGES_REQUESTED** [review]
Can you please make these changes @JUDOKICK ? Let me know you view.

And good work @JUDOKICK .

**Review by JUDOKICK — COMMENTED** [review]
Fixed as suggested!

**Review by itzmeanjan — APPROVED** [review]
Looks great +1

**Pull request merged**